[2022-12-26 15:16:35] [INFO    ] discord.client: logging in using static token
[2022-12-26 15:16:36] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 83185b06d790ee4db0bd568603884e67).
[2022-12-26 15:47:05] [ERROR   ] discord.ext.commands.bot: Ignoring exception in command plot
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/flags.py", line 581, in convert
    values = arguments[name]
KeyError: 'f'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1349, in invoke
    await ctx.command.invoke(ctx)
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1015, in invoke
    await self.prepare(ctx)
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/core.py", line 932, in prepare
    await self._parse_arguments(ctx)
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/core.py", line 847, in _parse_arguments
    kwargs[name] = await self.transform(ctx, param, attachments)
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/core.py", line 709, in transform
    return await run_converters(ctx, converter, argument, param)  # type: ignore
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/converter.py", line 1340, in run_converters
    return await _actual_conversion(ctx, converter, argument, param)
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/converter.py", line 1220, in _actual_conversion
    return await converter.convert(ctx, argument)
  File "/opt/homebrew/lib/python3.10/site-packages/discord/ext/commands/flags.py", line 584, in convert
    raise MissingRequiredFlag(flag)
discord.ext.commands.errors.MissingRequiredFlag: Flag 'f' is required and missing
[2022-12-26 16:36:35] [ERROR   ] discord.client: Attempting a reconnect in 1.97s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-26 16:36:37] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 83185b06d790ee4db0bd568603884e67.
[2022-12-26 16:52:52] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 16:52:58] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: e89ca01d68fa646f858cd170d83ab3eb).
[2022-12-26 16:58:16] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session e89ca01d68fa646f858cd170d83ab3eb.
[2022-12-26 17:00:33] [ERROR   ] discord.client: Attempting a reconnect in 1.74s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-26 17:00:35] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session e89ca01d68fa646f858cd170d83ab3eb.
[2022-12-26 17:09:29] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 17:09:34] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: af89094618ceda34a7d3524969f0a0f9).
[2022-12-26 17:26:02] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 17:26:07] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1b8ba7639f308059975a22820acc6014).
[2022-12-26 17:33:22] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 17:33:27] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: b24fe43eb0b96e11fe4a6d33b0316444).
[2022-12-26 17:36:27] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 17:36:32] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 97c0e4c82919437de6c2de57769ccd23).
[2022-12-26 17:38:18] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 97c0e4c82919437de6c2de57769ccd23.
[2022-12-26 17:42:32] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 97c0e4c82919437de6c2de57769ccd23.
[2022-12-26 17:43:42] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 97c0e4c82919437de6c2de57769ccd23.
[2022-12-26 17:45:41] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 97c0e4c82919437de6c2de57769ccd23.
[2022-12-26 17:47:17] [ERROR   ] discord.client: Attempting a reconnect in 3.38s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-26 17:47:21] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 97c0e4c82919437de6c2de57769ccd23.
[2022-12-26 17:52:56] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 17:53:01] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7639ad514d083f3bfcff32207e369651).
[2022-12-26 17:55:13] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 7639ad514d083f3bfcff32207e369651.
[2022-12-26 18:04:06] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 18:04:11] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: fe3d6cf8a273c2c63d150c2a1af06c3c).
[2022-12-26 18:21:45] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 18:21:51] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6a82980cbf45c1652e09492c7b52a363).
[2022-12-26 18:29:03] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 18:29:08] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: da9ceacdcacc5147bb5f777cf9a5a1d5).
[2022-12-26 18:42:56] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 18:43:01] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: dd36028dd5b3d38fa848094ca1588882).
[2022-12-26 18:45:50] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 18:45:55] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: c67a547af1cf48d4b46cf796d6cd7840).
[2022-12-26 18:56:13] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 18:56:19] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8d7fc852b47fc1003c7d6e5f741d5be6).
[2022-12-26 19:12:15] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 19:12:21] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0051e8495a96d43c42312c466a1705c7).
[2022-12-26 19:34:12] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 19:34:17] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6e1a30a154c0bdde52bbb0386d6f9e21).
[2022-12-26 19:42:04] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 19:42:09] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: f1556de03fcf3c5d8401581ff744f524).
[2022-12-26 19:43:18] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session f1556de03fcf3c5d8401581ff744f524.
[2022-12-26 19:59:41] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 19:59:46] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: b7a93175165935702f5c5060ed52ad4f).
[2022-12-26 20:07:59] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 20:08:05] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7434d43276f383e401c5c26ed41be354).
[2022-12-26 20:20:44] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 20:20:49] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1d82052400cd69ca8dff948976d13ac1).
[2022-12-26 20:22:40] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 1d82052400cd69ca8dff948976d13ac1.
[2022-12-26 20:44:49] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 20:44:55] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 9b2def13c1c013a62e3463dac97f408d).
[2022-12-26 20:52:24] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 20:52:29] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0a88dabd9103c9d362eba07d2062187b).
[2022-12-26 20:55:07] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 20:55:13] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1b9ec1f62b63acae28744d268976906c).
[2022-12-26 21:12:25] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 21:12:31] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 25cbe0245308c7d23cf8e77686b904ed).
[2022-12-26 21:29:17] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 21:29:22] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3d5f5316b60824dc41105718c53d7c94).
[2022-12-26 21:57:45] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 21:57:50] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 01410cc34ba717a90d43704a6b55af3e).
[2022-12-26 21:58:45] [ERROR   ] discord.client: Attempting a reconnect in 1.13s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-26 21:58:46] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 01410cc34ba717a90d43704a6b55af3e.
[2022-12-26 22:04:30] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 22:04:35] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 15ccf968749aab09c3083535d63b6a40).
[2022-12-26 22:06:04] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 15ccf968749aab09c3083535d63b6a40.
[2022-12-26 22:11:11] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 22:11:18] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: cdbce37240f293fbbed515f3aefa800e).
[2022-12-26 22:13:30] [ERROR   ] discord.client: Attempting a reconnect in 1.42s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-26 22:13:32] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session cdbce37240f293fbbed515f3aefa800e.
[2022-12-26 22:23:22] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 22:23:28] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 5e2d9a6e95a2cde1b9f0cb7f35dbe655).
[2022-12-26 22:26:45] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 5e2d9a6e95a2cde1b9f0cb7f35dbe655.
[2022-12-26 22:36:42] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 22:36:47] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: adc5be2eb5ef2fdd5f6199f264255509).
[2022-12-26 22:51:36] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 22:51:41] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 85a37bbb1ae3cef829464e9d44c4488e).
[2022-12-26 23:04:15] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 23:04:20] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8bfe2f0dd31eae469e2ebd61f55890cb).
[2022-12-26 23:26:13] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 23:26:19] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: ceebd9d2cc3e7524867c0a77f44bf565).
[2022-12-26 23:35:20] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 23:35:25] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 390285d4303a57be402a78e6a21d8e88).
[2022-12-26 23:38:27] [ERROR   ] discord.client: Attempting a reconnect in 1.30s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-26 23:38:28] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 390285d4303a57be402a78e6a21d8e88.
[2022-12-26 23:44:42] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 23:44:47] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 27e9a2ab058d5044959a8f59cc86ef64).
[2022-12-26 23:58:47] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-26 23:58:53] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7beff0435457f3c2cbdd135a2a03ff94).
[2022-12-27 00:04:12] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:04:17] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: be519ed44c2714ad512c2920716b9f13).
[2022-12-27 00:06:40] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:06:45] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 03960608af724035d4106cc19d55fee4).
[2022-12-27 00:08:27] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 03960608af724035d4106cc19d55fee4.
[2022-12-27 00:10:42] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:10:48] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 65ce7f49baae1fab895b91162b64f59d).
[2022-12-27 00:15:35] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:15:40] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 17101eb383a1f348e5bda7f0bba4b619).
[2022-12-27 00:18:51] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:18:56] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: e878f9583e02959fd88de72cb76003ec).
[2022-12-27 00:21:47] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session e878f9583e02959fd88de72cb76003ec.
[2022-12-27 00:24:14] [ERROR   ] discord.client: Attempting a reconnect in 15.52s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 00:24:30] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session e878f9583e02959fd88de72cb76003ec.
[2022-12-27 00:27:14] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:27:19] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6d324a12c3f96b7d89d9c2260b06201b).
[2022-12-27 00:29:31] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:29:36] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 60780a1252811c8dfeb5a04b0be39469).
[2022-12-27 00:33:33] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:33:38] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2f14d64e4da92f6abbd1a9753ab00878).
[2022-12-27 00:44:07] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:44:12] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7d68874dabc08c554c2f0f9f3ce87976).
[2022-12-27 00:49:01] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:49:06] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: a2c6888867903eb2d00833431ec4e6d8).
[2022-12-27 00:50:25] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session a2c6888867903eb2d00833431ec4e6d8.
[2022-12-27 00:51:44] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session a2c6888867903eb2d00833431ec4e6d8.
[2022-12-27 00:53:21] [ERROR   ] discord.client: Attempting a reconnect in 13.72s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 00:53:35] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session a2c6888867903eb2d00833431ec4e6d8.
[2022-12-27 00:54:30] [ERROR   ] discord.client: Attempting a reconnect in 57.46s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 00:55:55] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 00:56:00] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 70bbdf41e4ca5907b758f882dcc9808c).
[2022-12-27 00:57:43] [ERROR   ] discord.client: Attempting a reconnect in 8.83s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 00:57:52] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 70bbdf41e4ca5907b758f882dcc9808c.
[2022-12-27 01:04:56] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 01:05:01] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 4720e6577c341c41e6614c787895d1e9).
[2022-12-27 01:10:25] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 01:10:30] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 34b1d69623dd6369cad09101fa40550b).
[2022-12-27 01:23:09] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 01:23:14] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1ed4215ed6779fec80efb119d815515a).
[2022-12-27 01:31:27] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 01:31:32] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8e848b85c0d6cde5f0acd3fb4a2ef954).
[2022-12-27 01:34:11] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 8e848b85c0d6cde5f0acd3fb4a2ef954.
[2022-12-27 01:47:27] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 01:47:33] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: d7e2d73ff114d85a45acfc307b89a492).
[2022-12-27 01:51:50] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 01:51:55] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0e646c0ab6bea5b30ac7dc0166b2f251).
[2022-12-27 02:07:49] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 02:07:54] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: ca7c194ed9555baca98a3c0e931945c1).
[2022-12-27 02:16:24] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 02:16:29] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 289b484a45e1141f3c8af861ee2bf0e5).
[2022-12-27 02:32:42] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 02:32:47] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 20ccadacb7200d98a4c61c6dbc219ac0).
[2022-12-27 02:35:00] [ERROR   ] discord.client: Attempting a reconnect in 159.54s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 02:58:33] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 02:58:38] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: f06acaa7da4b0ed645411cbc9ea8dea9).
[2022-12-27 03:18:25] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 03:18:31] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 71e83e327e737a47e337d9f2390728ba).
[2022-12-27 03:35:36] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 03:35:41] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 68796be6e2582e1ec669c5c1869b1bba).
[2022-12-27 03:52:45] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 03:52:50] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: d967098eee38b7bf9f72548b01a6dc89).
[2022-12-27 04:09:23] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 04:09:28] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1b7046e30d0c27f6ce3db20d82048e9a).
[2022-12-27 04:25:23] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 04:25:29] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: eb9a7687b84cfe56e0e18e70e76e0cbc).
[2022-12-27 04:34:39] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 04:34:44] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: c1388376d7fe6937b38332382c548a07).
[2022-12-27 04:50:40] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 04:50:45] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 93ef6aca1ffed13f4914d44bc3c1dfd4).
[2022-12-27 05:06:43] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 05:06:48] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 80db4f6864b5b54d9c6e6b9a16a7c43a).
[2022-12-27 05:25:05] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 05:25:11] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 23b71bfd55897f6a486915ed03a1c9af).
[2022-12-27 05:36:10] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 05:36:15] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: eb65dd99985264e20b35c4eb0140c4a5).
[2022-12-27 05:37:10] [ERROR   ] discord.client: Attempting a reconnect in 127.79s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 06:12:10] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 06:12:15] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 088cfc2c19a3c469da7e55ea15a646bf).
[2022-12-27 06:30:11] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 06:30:17] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 7eb2b25088a45483d395caf5740e4dcd).
[2022-12-27 06:34:14] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 06:34:20] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 6745f049e037b7db9d89c13400f19292).
[2022-12-27 06:51:58] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 06:52:03] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: c83beb63fc843568e5f94bfdcc23995a).
[2022-12-27 07:00:19] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 07:00:24] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: c2798b30b974fd3c96e8e840f08a8878).
[2022-12-27 07:19:38] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 07:19:44] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: d678bf3cd4f92879d250416df719a7fa).
[2022-12-27 07:33:45] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 07:33:50] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: b114ebdb3e07f069428654c2a8a47b7d).
[2022-12-27 07:51:20] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 07:51:26] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 37131f94837246a3f63b97d4e68a3db9).
[2022-12-27 08:06:07] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 08:06:12] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 963a3ab2a87b324e3ff6ccf5dea9b9dd).
[2022-12-27 08:08:24] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 08:08:29] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 16b2100ef5a5dbf5dc9a8c4a8d96c261).
[2022-12-27 08:11:53] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 08:11:59] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 406d6611e70874cc8a8838ba82e1aace).
[2022-12-27 08:29:28] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 08:29:33] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 5d76a9081e5ef0b51ae83f20e824828f).
[2022-12-27 08:37:48] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 08:37:54] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 81d0022b4a7edb9d77f16bed154b6add).
[2022-12-27 08:54:30] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 08:54:36] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: f99d9171a2dec095c302ab5e009eb63a).
[2022-12-27 09:06:20] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 09:06:25] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: fcc4b22575151c80aa55b715c5d5dc8a).
[2022-12-27 09:20:48] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 09:20:53] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1f9472854b7005ad4db0fb1945ba6bba).
[2022-12-27 09:38:05] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 09:38:11] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 025c52a8a6b5514569aecb2f762d33eb).
[2022-12-27 09:40:35] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 025c52a8a6b5514569aecb2f762d33eb.
[2022-12-27 09:46:01] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 09:46:07] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: acc81336af0fa4c7d9cfa7126cad4644).
[2022-12-27 10:01:53] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 10:01:58] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: bb007b33cb54029db21326dd3b83c940).
[2022-12-27 10:18:45] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 10:18:51] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3df566dc6704c426663766302897bf1c).
[2022-12-27 10:34:37] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 10:34:42] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 201390ed01690f41886a552090272c47).
[2022-12-27 10:35:37] [ERROR   ] discord.client: Attempting a reconnect in 252.99s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 11:01:08] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 11:01:13] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: c715f7384abae0ba3d7ffba38ee9ded1).
[2022-12-27 11:05:27] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 11:05:32] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2bee26bd6a86ce1983a8748f3b242bed).
[2022-12-27 11:08:58] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 11:09:03] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3550238bd08ca21173106c9a5bf88a7d).
[2022-12-27 11:10:31] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 3550238bd08ca21173106c9a5bf88a7d.
[2022-12-27 11:12:40] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 11:12:46] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 05b471c607b1bf58257da2fb1b60ac27).
[2022-12-27 11:15:33] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 11:15:38] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 9e80173fa868e5656c5a5713b4697898).
[2022-12-27 11:16:36] [ERROR   ] discord.client: Attempting a reconnect in 578.39s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-27 12:04:29] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 12:04:35] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2e9d4e9729453f7e4a0c43a3b85cc9ea).
[2022-12-27 12:28:06] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 12:45:52] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 12:47:45] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 13:12:31] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 13:12:49] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: e26a40a265695084ef43e0f1752d8af5).
[2022-12-27 13:42:52] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 13:59:40] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 13:59:45] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 970442b7f3a9bb5386d8500b32bafd49).
[2022-12-27 14:31:51] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 14:31:56] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 24a0fa2ab105c627e5a92a18086e0f0b).
[2022-12-27 15:21:18] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 15:21:24] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 947ebf5794af1c13402b172df8e786ad).
[2022-12-27 15:53:04] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 15:53:09] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 36b5f2fbe65e9c6dda997bd964bd81b2).
[2022-12-27 16:27:49] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 16:27:54] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 21abf4763680bab93358182f7fd8dc7c).
[2022-12-27 17:16:38] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 17:24:37] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 17:26:22] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 17:42:11] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 17:42:16] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: a2086f0db7e1a7d6783e506974f4c149).
[2022-12-27 18:24:57] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 18:25:30] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 5d0185c92656591ff3ab84c591121249).
[2022-12-27 18:31:02] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 18:31:08] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8878f8183904cad8d5363f34e9dd1827).
[2022-12-27 18:59:37] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 19:00:05] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 292ae4ebffa16274e9d1b5c5a95193ea).
[2022-12-27 19:57:30] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 20:13:48] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 20:13:54] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: e7a2516e413aadc39b7bd2d98be6b430).
[2022-12-27 21:08:20] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 21:24:40] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 21:41:55] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 21:42:02] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 9bc5a4746c4ae051ed5d89ac2e17d6f9).
[2022-12-27 22:04:32] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 22:21:14] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 22:36:36] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 22:36:41] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: a5a6a2d8e6ebab499ad250e117877ee6).
[2022-12-27 23:09:05] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 23:23:16] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-27 23:23:22] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 91808889f0c6bcfc5c0535b42eb99ea1).
[2022-12-28 00:15:19] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 00:15:41] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 4079c6df65cbb78cb9356a19943499ee).
[2022-12-28 00:52:50] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 01:00:23] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 01:18:09] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 01:18:14] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: dfbd8ad9c91ed733a8fb8c1af54348fb).
[2022-12-28 01:40:55] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 01:41:00] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 1d4b8b12c3a42d950aae1afd6f860658).
[2022-12-28 02:23:44] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 02:40:53] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 02:41:05] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: ad63463bcce39475e8a9f736643e7a46).
[2022-12-28 03:53:36] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 03:53:42] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 401dd7af353ff59f56a1792c073fe469).
[2022-12-28 04:54:36] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 05:10:44] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 05:27:40] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 05:45:31] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 05:45:36] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 83f9767df77c516007c68e994cb015b1).
[2022-12-28 06:51:51] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 07:00:04] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 07:08:44] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 07:08:50] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8cd766d8479355635ff4ec277a0f5cce).
[2022-12-28 08:09:42] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 08:14:51] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 08:22:47] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 08:40:49] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 08:40:55] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0c29fcc017707431f24daa40e7cdeb46).
[2022-12-28 09:20:25] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 09:20:51] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: af4e3c6460fd01f606dca166551e6eac).
[2022-12-28 09:57:24] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 09:57:29] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: c9f6ca5d6c69332c732ffc543b8331d2).
[2022-12-28 10:10:31] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 10:27:47] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 10:43:05] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 10:43:10] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 04f7e4cd6699675d30a8aa16fe2a756f).
[2022-12-28 11:18:02] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 11:21:12] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 11:27:34] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 11:27:39] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: fd50e76b25338afd55a10904e4d5a12d).
[2022-12-28 11:58:00] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 12:02:21] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 12:02:26] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: c425403b3ec6d81f05889201bf678e93).
[2022-12-28 12:03:24] [ERROR   ] discord.client: Attempting a reconnect in 720.96s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 13:21:43] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 13:21:48] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 83505b9a7d9990abf130db30c69b3517).
[2022-12-28 13:23:19] [ERROR   ] discord.client: Attempting a reconnect in 603.10s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 13:40:31] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 13:40:37] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 5d130a2d0ef39ffca9e9f937ac74f754).
[2022-12-28 13:42:06] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 5d130a2d0ef39ffca9e9f937ac74f754.
[2022-12-28 13:43:02] [ERROR   ] discord.client: Attempting a reconnect in 928.70s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 14:39:59] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 14:40:04] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: d86b985d99f80970328b7ff6b3d4738f).
[2022-12-28 14:44:01] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 14:50:03] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 14:50:15] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 36683e36af8a51989acc36439e6716ea).
[2022-12-28 14:51:49] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 36683e36af8a51989acc36439e6716ea.
[2022-12-28 14:54:07] [ERROR   ] discord.client: Attempting a reconnect in 28.95s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 14:56:29] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 14:56:35] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 62a047bcd9b2ff98b7a9ff50c845f938).
[2022-12-28 14:57:30] [ERROR   ] discord.client: Attempting a reconnect in 88.40s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 15:07:43] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 15:07:48] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 021e9f8cef6d340bb887a15b41326359).
[2022-12-28 15:10:08] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 021e9f8cef6d340bb887a15b41326359.
[2022-12-28 15:12:39] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 021e9f8cef6d340bb887a15b41326359.
[2022-12-28 15:15:45] [ERROR   ] discord.client: Attempting a reconnect in 328.69s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 15:23:06] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 15:23:12] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 677f67824a57853a0a3d45e483411fd1).
[2022-12-28 15:24:43] [ERROR   ] discord.client: Attempting a reconnect in 979.05s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 15:45:41] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 15:45:47] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 3694b8b5656681ac90bb70403bd6fb0c).
[2022-12-28 15:46:42] [ERROR   ] discord.client: Attempting a reconnect in 8.06s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 15:46:51] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 3694b8b5656681ac90bb70403bd6fb0c.
[2022-12-28 15:49:08] [ERROR   ] discord.client: Attempting a reconnect in 343.61s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 16:16:58] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 16:17:04] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: a801fb22eecd1e90fec03dabc2a0b85c).
[2022-12-28 16:22:08] [ERROR   ] discord.client: Attempting a reconnect in 131.69s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 16:24:47] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 16:24:52] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8464fd64d6a836234218f2acdb79ec60).
[2022-12-28 16:27:25] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 8464fd64d6a836234218f2acdb79ec60.
[2022-12-28 16:30:24] [ERROR   ] discord.client: Attempting a reconnect in 524.40s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 16:41:24] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 16:41:29] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 8e4a899df2f1742e55683edad0ca95b1).
[2022-12-28 16:43:00] [ERROR   ] discord.client: Attempting a reconnect in 477.85s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 16:52:28] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 16:52:34] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: eb02184862d001bacc8e63903f1e8a62).
[2022-12-28 16:53:29] [ERROR   ] discord.client: Attempting a reconnect in 262.92s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 16:58:51] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 16:58:57] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: b7b241df44670d353a377a96970d2fa3).
[2022-12-28 17:01:09] [ERROR   ] discord.client: Attempting a reconnect in 742.22s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 17:15:50] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 17:15:55] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: ae9608f69fbc07e2ba50faa5ddbf0935).
[2022-12-28 17:16:50] [ERROR   ] discord.client: Attempting a reconnect in 1010.56s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 17:37:57] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 17:38:03] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: b2dbf1d2e6352709cd37b2f8c0997346).
[2022-12-28 17:40:15] [ERROR   ] discord.client: Attempting a reconnect in 997.41s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 18:01:17] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 18:01:22] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: a45284b3b22b9c880e3287a3518574f3).
[2022-12-28 18:03:36] [ERROR   ] discord.client: Attempting a reconnect in 716.35s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 18:18:16] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 18:18:21] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 869a154d6d2d520b9d9d71e6ce3211ee).
[2022-12-28 18:19:16] [ERROR   ] discord.client: Attempting a reconnect in 838.98s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 18:36:39] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 18:36:44] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 035da9d26e1b2253625765519702ca5e).
[2022-12-28 18:37:39] [ERROR   ] discord.client: Attempting a reconnect in 344.41s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 18:44:21] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 18:44:26] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 584ed236e6e0c046ebea84d26323711b).
[2022-12-28 18:45:21] [ERROR   ] discord.client: Attempting a reconnect in 44.89s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 18:46:06] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 584ed236e6e0c046ebea84d26323711b.
[2022-12-28 18:48:03] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 584ed236e6e0c046ebea84d26323711b.
[2022-12-28 18:50:36] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 584ed236e6e0c046ebea84d26323711b.
[2022-12-28 18:53:15] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 584ed236e6e0c046ebea84d26323711b.
[2022-12-28 18:55:53] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 584ed236e6e0c046ebea84d26323711b.
[2022-12-28 18:58:22] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 584ed236e6e0c046ebea84d26323711b.
[2022-12-28 19:00:55] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 584ed236e6e0c046ebea84d26323711b.
[2022-12-28 19:04:56] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 19:05:02] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 823788aef8bab65d2fa442811484277b).
[2022-12-28 19:06:35] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 823788aef8bab65d2fa442811484277b.
[2022-12-28 19:08:53] [ERROR   ] discord.client: Attempting a reconnect in 563.53s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 19:21:55] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 19:22:01] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 2608faa6dc36cafdeea4945b7c577247).
[2022-12-28 19:23:10] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 2608faa6dc36cafdeea4945b7c577247.
[2022-12-28 19:25:48] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 2608faa6dc36cafdeea4945b7c577247.
[2022-12-28 19:31:10] [ERROR   ] discord.client: Attempting a reconnect in 88.96s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 19:32:39] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 19:32:44] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 0c2b7892d4040ad8c5b8e6f04e054ccc).
[2022-12-28 19:33:39] [ERROR   ] discord.client: Attempting a reconnect in 853.20s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 19:50:29] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 19:50:35] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: dfcd50e05e7a85075200fd19e4367036).
[2022-12-28 19:51:46] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session dfcd50e05e7a85075200fd19e4367036.
[2022-12-28 19:54:23] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session dfcd50e05e7a85075200fd19e4367036.
[2022-12-28 19:56:56] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session dfcd50e05e7a85075200fd19e4367036.
[2022-12-28 19:59:13] [ERROR   ] discord.client: Attempting a reconnect in 750.76s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 20:14:06] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 20:14:11] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: b64979b2df644f6f4bb02c17c77b50f0).
[2022-12-28 20:15:06] [ERROR   ] discord.client: Attempting a reconnect in 423.13s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 20:23:50] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 20:23:56] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 49123c92a4768b4564d5c0697d6c34cb).
[2022-12-28 20:25:47] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 49123c92a4768b4564d5c0697d6c34cb.
[2022-12-28 20:28:25] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 49123c92a4768b4564d5c0697d6c34cb.
[2022-12-28 20:30:46] [ERROR   ] discord.client: Attempting a reconnect in 725.31s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 20:45:37] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 20:45:42] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 958a1212df09e62bb198c05e4e80530f).
[2022-12-28 20:46:37] [ERROR   ] discord.client: Attempting a reconnect in 794.28s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 21:03:12] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 21:03:17] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 52e97c5af0c3566dcd556e60851f6c4a).
[2022-12-28 21:04:48] [ERROR   ] discord.client: Attempting a reconnect in 590.31s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 21:17:49] [INFO    ] discord.gateway: Shard ID None session has been invalidated.
[2022-12-28 21:17:54] [INFO    ] discord.gateway: Shard ID None has connected to Gateway (Session ID: 82aad8b6eaca530001e87e59164a262e).
[2022-12-28 21:18:49] [ERROR   ] discord.client: Attempting a reconnect in 32.46s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[2022-12-28 21:19:22] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 82aad8b6eaca530001e87e59164a262e.
[2022-12-28 21:21:18] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 82aad8b6eaca530001e87e59164a262e.
[2022-12-28 21:23:56] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 82aad8b6eaca530001e87e59164a262e.
[2022-12-28 21:26:34] [INFO    ] discord.gateway: Shard ID None has successfully RESUMED session 82aad8b6eaca530001e87e59164a262e.
[2022-12-28 21:28:55] [ERROR   ] discord.client: Attempting a reconnect in 932.67s
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/discord/client.py", line 627, in connect
    await self.ws.poll_event()
  File "/opt/homebrew/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
